pkg/serverconfig/types.go

```diff
@@ -129,3 +129,9 @@ type HelmChartRepo struct {
 type Helm struct {
 	ChartRepo HelmChartRepo `yaml:"chartRepository"`
 }
+
+type ManagedClusterConfig struct {
+	Name   string `json:"name" yaml:"name"`
+	Server string `json:"server" yaml:"server"`
+	CAData string `json:"caData" yaml:"caData"`
```

Comment:
We might need multiple CA files. Maybe make `apiServer` it its own stanza so we can add other servers and CA files?

```
{
  name,
  apiServer: {
    url,
    caData,
  },
}
```